audioread/gstdec.py

```diff
@@ -47,9 +47,11 @@
 """
 from __future__ import with_statement
 
-import gst
+import gi
+gi.require_version('Gst', '1.0')
```

Comment:
Maybe this test (and `Gst.init(None)`, below) should be done on-demand, so they can't fail after the `_gst_available` check succeeds?